src/client/Microsoft.Identity.Client/Internal/Requests/DeviceCodeRequest.cs

```diff
@@ -29,7 +29,7 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
         {
             await ResolveAuthorityAsync().ConfigureAwait(false);
 
-            var client = new OAuth2Client(ServiceBundle.ApplicationLogger, ServiceBundle.HttpManager);
+            var client = new OAuth2Client(ServiceBundle.ApplicationLogger, ServiceBundle.HttpManager, null);
```

Comment:
I see that there are a number of instances where this constructor is called with the binding cert `null`. Wondering if it makes sense to add an overload where it is assumed to be null? Or maybe a default value?